pkgs/development/ocaml-modules/zmq/default.nix

```diff
@@ -18,9 +18,10 @@ stdenv.mkDerivation rec {
     sha256 = "1f5l4bw78y4drabhyvmpj3z8k30bill33ca7bzhr02m55yf6gqpf";
   };
 
-  patches = [
-    ./ocaml-zmq-issue43.patch
-  ];
+  patches =
+    if builtins.compareVersions stdint.version "0.5.1" == -1 then
+      [ ./ocaml-zmq-issue43.patch ]
+    else [];
```

Comment:
Please remove the patch, as it is no longer needed.

Reply:
The patch file should be removed too.

Reply:
Oh well, sorry that I need to be micromanaged like that :)